changes/README.md

```diff
@@ -0,0 +1,2 @@
+## Need to add some directions on how the updated process works
+
```

Comment:
Do you want to add those directions now?

Reply:
Talked to Andrew about this.  Someone can add the details when working on a subsequent PR.